app/src/test/java/org/simple/clinic/scanid/ScanSimpleIdEffectHandlerTest.kt

```diff
@@ -234,14 +234,19 @@ class ScanSimpleIdEffectHandlerTest {
         value = "91704c43-c35f-41d2-9887-d5cb6e580dd9",
         type = IndiaNationalHealthId
     )
+    val indiaNationalHealthID = "12341644123412"
+    val indiaNHIDInfoPayload = TestData.indiaNHIDInfoPayload(
+        healthIdNumber = indiaNationalHealthID
+    )
+    val patientPrefillInfo = indiaNHIDInfoPayload.toPatientPrefillInfo()
```

Comment:
We can probably add a new `TestData` method to create the `PatientPrefillInfo`, wdyt?

Reply:
Yes, seems like a good idea